rust/src/utils.rs

```diff
@@ -158,10 +158,36 @@ pub fn varsubst(instr: &str, vars: &HashMap<String, String>) -> Fallible<String>
     return Ok(s);
 }
 
+/// Quote a string using C-style escapes; intended to be decoded by
+/// systemd in places like systemd-tmpfiles which uses `EXTRACT_QUOTES`.
+pub fn c_quote(instr: &str) -> String {
```

Comment:
We already have `rpmostree_maybe_shell_quote` which I think does the same thing, right?